capstone_project/src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -28,52 +37,61 @@
 @WebServlet("/school-data")
 public class MapServlet extends HttpServlet {
 
-  private ArrayList<School> schools = new ArrayList<School>();
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Shouldn't doGet and doPost require authentication?

Reply:
@andrewnguyen I'm not sure what you mean? We weren't planning to have users login in order to use this feature.

Reply:
@andrewnguyen fixed!